### DISCUSSION TITLE
Add the ability to name bookmarks with labels

### DISCUSSION THREAD

**Discussion by Maxylan** [ppy/osu#12978] [Ideas]
### Being able to "name" the bookmarks (blue lines) in the editor with labels would be an easy, small thing to add, that would go a long way towards keeping the editor and the work you do in it more organized.

How would the editor be more organized? Well currently bookmarks are multipurpose, with each beatmapper using them for different reasons like to mark strong sounds, sections that change or sections for collabs. 
**Labels would still allow bookmarks to be multipurpose, with the added benefit of less confusion**, and anyone other than the creator looking at these bookmarks no longer have to guess what the bookmarks are for.
**Labeled bookmarks could also potentially be implemented into ModdingV2**, any bookmarks in the editors timeline that are labeled would be visible in the modding V2 timeline for each difficulty, helping modders in more ways than one.

**Example of how this could be done**, is to add a tab in the "song options" that displays all current bookmarks and their time signatures, where each bookmark would have an input field below it where you could give it a name/label. Labels the user assigned to a bookmark would be visible if, let's say a user hovers over the bookmark with their mouse.

**Comment by Bu3nds** [comment]
I would love this!
